src/esn.calendar.libs/app/components/attendee/attendee-item/attendee-item.pug

```diff
@@ -10,7 +10,7 @@
       )
         | {{ctrl.attendee.displayName}} 
     .meta
-      a(ng-href="mailto:{{ ::ctrl.attendee.email }}" op-inbox-compose op-inbox-compose-users="[ctrl.attendee]" op-inbox-compose-subject="{{::ctrl.subject}}")
+      a(ng-href="{{$ctrl.mailtoURL}}:{{ ::ctrl.attendee.email }}" op-inbox-compose op-inbox-compose-users="[ctrl.attendee]" op-inbox-compose-subject="{{::ctrl.subject}}")
```

Comment:
how does this work? if the mailto URL is `http://mailto.example.com` and the email `test@a.com` then this `ng-href` will become `ng-href="http://mailto.example.com:test@a.com"` which will not work I guess.

if I remember correctly there is an interceptor for the `mailto:` URI that redirects you to the SPA. it should be somewhere in the inbox or mailto SPA.

Reply:
the mailto url will be configurable in openpaas js http://mailto.open-paas.org.local/mailto/